libs/data-types/src/lib/utilities/getCoscradDataSchema.ts

```diff
@@ -1,6 +1,11 @@
+import { UnionDataTypeDefinition } from '../types';
 import { ClassSchema } from '../types/ClassSchema';
 import getCoscradDataSchemaFromPrototype from './getCoscradDataSchemaFromPrototype';
 
+export interface UnionFinderService {
+    find(name: string): UnionDataTypeDefinition;
+}
+
 // eslint-disable-next-line
 export default (TargetClass: Object): ClassSchema<Record<string, unknown>> =>
```

Comment:
We really need to sort out the types that are used in this lib and their relationship to the interfaces in `@coscrad/api-interfaces`.